packages/flutter/test/painting/image_stream_test.dart

```diff
@@ -107,7 +104,9 @@ void main() {
     await tester.idle();
     expect(mockCodec.numFramesAsked, 0);
 
-    void listener(ImageInfo image, bool synchronousCall) { }
+    void listener(ImageInfo image, bool synchronousCall) {
+      addTearDown(image.dispose);
+    }
     imageStream.addListener(ImageStreamListener(listener));
```

Comment:
should we also remove this listener at the end of the test?

Reply:
For some reasons this one does not cause leaks if not removed.